File: official/guides/customs-guide/golang/create-customs-item.go
customsItem, err := client.CreateCustomsItem(
  &easypost.CustomsItem{
    Description:    "T-shirt",
    Quantity:       1,
    Weight:         6,
    Value:          11,
    OriginCountry:  "US",
    HSTariffNumber: "610910",
  },
)

File: official/guides/customs-guide/node/create-customs-item.js
const customsItem1 = new api.CustomsItem({
    description: 'T-shirt',
    quantity: 1,
    value: 11,
    weight: 6,
    hs_tariff_number: '610910',
    origin_country: 'US',
});

customsItem1.save().then(console.log);
